charts/capsule/values.yaml

```
@@ -77,6 +77,21 @@ podAnnotations: {}
 # -- Set the priority class name of the Capsule pod
 priorityClassName: '' # system-cluster-critical
 
+# -- Set the securityContext for the Capsule pod
+podSecurityContext:
```

Add `seccompProfile` by default


```suggestion
podSecurityContext:
  seccompProfile:
    type: "RuntimeDefault"

```